lib/cc/service/http.rb

```diff
@@ -15,7 +15,23 @@ def default_http_options
     end
   end
 
-  def http_get(url = nil, params = nil, headers = nil)
+  def get(url = nil, body = nil, headers = nil, &block)
+    raw_get(url, body, headers, &block)
+  end
+
+  def post(url, body = nil, headers = nil, &block)
+    block ||= lambda{|*args| Hash.new }
+    response = raw_post(url, body, headers)
+    {
+      ok: response.success?,
+      params: body.as_json,
+      endpoint_url: url,
+      status: response.status,
+      message: "Success"
+    }.merge(block.call(response))
```

Comment:
After looking at the code more, I now see that `post` returns a hash but `get` returns, as it did before, some kind of http response object. I think this mismatch in the API is confusing, which is why I'm trying to explore if there's a better place for "response generation" that exists outside of this class. See main thread.